<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>genlicenseClass</class>
 <widget class="QWidget" name="genlicenseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>genlicense</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxTM">
       <property name="text">
        <string>时间限制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditStart"/>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditEnd"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnHostId">
       <property name="text">
        <string>选择HostId</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLicense">
       <property name="text">
        <string>生成license</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnSelLic">
       <property name="text">
        <string>选择License</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>明文</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_plaintext"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_plaintext_ciphertext">
       <property name="text">
        <string>明文转密文</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>密文</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ciphertext"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ciphertext_plaintext">
       <property name="text">
        <string>密文转明文</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="genlicense.qrc"/>
 </resources>
 <connections/>
</ui>
